pkg/charts/charts.go

```diff
@@ -22,14 +23,16 @@ type Chart struct {
 	Namespace string
 	RawValues []byte
 	Files     []*chartutil.BufferedFile
+	Fs        http.FileSystem
```

Comment:
It looks like Fs and Dir are always combined.  Any reason we can't just turn the `Dir` field into a `http.FileSystem` so that we don't need both?  (I think `os.File` would probably be better, but that's a larger change)

Reply:
I think its better this way, as not all code paths have to configure the filesystem field, they just use `static.Templates` which is the default i.e `/charts` and set the `Dir` based on their chart name.

If we combine them, Every `Chart` creator code-path will have to configure the `http.FileSystem` field to their chart directory. WDYT?